packages/core/utils/src/common/define-config.ts

```diff
@@ -117,6 +117,13 @@ export function defineConfig(config: Partial<ConfigModule> = {}): ConfigModule {
     ...config.modules,
   }
 
+  // Remove any modules set to false
```

Comment:
Although I approved, I think it might be good to already look into adding these defaults in the form of a `template` rather than having them hard-coded in `define-config`.